platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,119 @@
+# Platform Design Proposal Process
+
+This document describes the process for creating and reviewing platform design proposals.
+
+The status of all active design proposals can be viewed
+at [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project board.
+
+---
+
+## Design Proposal Process Flow
+
+![](designProposalFlow.svg)
+
+## Creating A Proposal
+
+All proposals have an owner who is responsible for taking it through the process to completion. Proposals must be
+presented in the form of a PR containing the proposal in a subdirectory of `platform-sdk/docs/proposals/`. Proposals may
+contain diagrams and supporting materials. Diagrams must be in SVG format and all content must be in an editable
```

Comment:
Can we define that we use draw.io for all diagrams?

Reply:
sure